config/locales/simple_form.oc.yml

```diff
@@ -12,6 +12,8 @@ oc:
         data: Fichièr CSV exportat d’una autra instància Mastodon
       sessions:
         otp: Picatz lo còdi d’autentificacion en dos temps (Two factor code) de vòstre mobil o utilizatz un de vòstres còdis de recuperacion.
+  user:
```

Comment:
@Quent-in 5f9cb4888298528afc4866d5dc72c43877a41305 is the difference that makes similar changes in Japanese. please refer.